Python/Product/Analysis/Analyzer/InterpreterScope.cs

```diff
@@ -246,7 +246,7 @@ internal bool RemoveVariable(string name, out VariableDef value)
         }
 
         internal virtual void ClearVariables() {
-            _variables = new AnalysisDictionary<string, VariableDef>();
+            _variables.Clear();
```

Comment:
Not sure that we should be reverting back to this? Sure, it'll only affect module-level variables, but those are important too :)